src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $field-gap;
+
+  width: $width-of-calendar-sheet;
+
+  &__date {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-box-size;
+    height: $day-box-size;
```

Comment:
u can create mixin for these properties. Up to you)

Reply:
rebuild wit mixins